src/CoreArbiterServerMain.cc

```diff
@@ -106,7 +106,7 @@ parseOptions(int* argcp, const char** argv) {
 }
 
 int main(int argc, const char** argv) {
-    Logger::setLogLevel(CoreArbiter::ERROR);
+    Logger::setLogLevel(CoreArbiter::NOTICE);
```

Comment:
This should probably be in its own commit. It is unrelated to other changes in this PR.